pkg/executables/dockerlinux.go

```diff
@@ -34,6 +34,10 @@ func (e *linuxDockerExecutable) ExecuteWithEnv(ctx context.Context, envs map[str
 	return e.Command(ctx, args...).WithEnvVars(envs).Run()
 }
 
+func (e *linuxDockerExecutable) ExecuteWithStdinAndEnv(ctx context.Context, in []byte, envs map[string]string, args ...string) (stdout bytes.Buffer, err error) {
```

Comment:
Why do we need this?
The purpose of the composable command interface is to avoid adding a ton of methods to the interface